### PR TITLE
Added translation for LANGUAGE DE

### PR DESCRIPTION
**Pull request by alfrede**
I have added some Translations, later i will go through all fields. (this is just a test for me)

### CODE REVIEW

**Comment by alfrede** [comment]
I have added some Translations, later i will go through all fields. (this is just a test for me)

**Comment by tijuca** [comment]
Why did you remove my name? Just add your name would be the correct thing.

**Comment by ciampix** [comment]
This is not the right place for tests @alfrede! Either set it right, especially the copyright note or we have to close it. @tijuca please verify the contents as ich spreche kein deutsch, natürlich  ...

**Comment by alfrede** [comment]
Hi Tijuca, sorry for that it was not my intention. But i found how this could be. In the Editorprogram (poedit)  as you see in the screenshot it tooks the name which you give there and fill it in the translation without any Information for me as a user. I hope it's okay if not then please suggest me an alternative solution. 
bildschirmfoto vom 2019-02-11 18-46-38

File: src/getting_started_in_kicad/po/addendum.de [hunk 1]
```diff
@@ -1,7 +1,9 @@
 PO4A-HEADER: mode=after; position=^\[\[contributors\]\]; beginboundary=\[\[
 [[translation]]
-*Übersetzung*
+//Translators translate the word just below and delete this comment
+*translation*
 
 //Translators put your names below here in the addendum file
-Carsten Schönert <c.schoenert@t-online.de>, 2016.
+Io <io@miosito.it>, year,year.
+Moi <moi@moi.fr>, year-interval.
```
Comment (alfrede) [inline src/getting_started_in_kicad/po/addendum.de]: Mathias Neumann <alfrede@opferindustries.de> 2019

**Comment by alfrede** [comment]
> This is not the right place for tests @alfrede! Either set it right, especially the copyright note or we have to close it. @tijuca please verify the contents as ich spreche kein deutsch, natürlich ...

Hi ciampix can we add a test repo for people like me who does this all for the first time? I just want to help the project and of course the production system is not the right place for testing. I will do my best to learn from my mistake and will not do the same.

**Comment by tijuca** [comment]
You need to distinguish between the setup within your preferred Po editor and the file `src/getting_started_in_kicad/po/addendum.de`. These are two separated things. The former is taken in account just to mark the author within the modified *.po file(s).

The file `src/getting_started_in_kicad/po/addendum.de` is taken dynamically within the build process of the documentation for Getting Started in KiCad. You will find the output right in the head of the file before the first sentence. Have a closer look at **Übersetzung** in the German GSIK files.
So deleting my or any other name here is just wrong as this cut of people who have contributed.

The translation you have done is fine, you will need to just revert the current changes in `src/getting_started_in_kicad/po/addendum.de`. And add you name under the existing name. I would do it this way.
```
$ git diff
diff --git a/src/getting_started_in_kicad/po/addendum.de b/src/getting_started_in_kicad/po/addendum.de
index ec7e66cd..a72c2b99 100644
--- a/src/getting_started_in_kicad/po/addendum.de
+++ b/src/getting_started_in_kicad/po/addendum.de
@@ -3,5 +3,6 @@ PO4A-HEADER: mode=after; position=^\[\[contributors\]\]; beginboundary=\[\[
 *Übersetzung*
 
 //Translators put your names below here in the addendum file
-Carsten Schönert <c.schoenert@t-online.de>, 2016.
+Carsten Schönert <c.schoenert@t-online.de>, 2016;
+Mathias Neumann <alfrede@opferindustries.de>, 2019.

```

**Comment by tijuca** [comment]
> Hi ciampix can we add a test repo for people like me who does this all for the first time? I just want to help the project and of course the production system is not the right place for testing. I will do my best to learn from my mistake and will not do the same.

There is no need for that. You have started a pull request so nothing will happen until someone is accepting it.
If you have some more questions about how to do things we have a mailing list. ;)
The rest is testing your pull request again and again before starting it finally. All you need for testing you can do locally.

**Comment by alfrede** [comment]
Okay thanks for your answers :+1: , i will do more translations (this time without deleting your name) :)

**Comment by ciampix** [comment]
Since we are near the 5.2 deadline, I'll merge this and fix the addendum myself. Thanks for the translation @alfrede and for your "coaching" @tijuca !

**Pull request merged**